### PR TITLE
Register form styling and functionally

### PR DESCRIPTION
**Pull request by Vansers**
Let me know what you think, it's just a start.

### CODE REVIEW

File: app/Http/Controllers/Auth/RegisterController.php [hunk 1]
```diff
@@ -56,26 +65,34 @@ protected function validator(array $data)
...
+        $this->validate(request(), [
+            'name' => 'required',
+            'email' => 'required|email',
+            'airline' => 'required',
+            'home_airport' => 'required',
+            'password' => 'required|confirmed'
         ]);
 
-        $role = Role::where('name', 'admin')->first();
+        # TODO: I'm just feeling we need to do something with Ranking? I forgot.
```
Comment (nabeelio) [inline app/Http/Controllers/Auth/RegisterController.php]: Let's move this into the `Services/PilotService.php` class. After creating the user, you can call `calculatePilotRank($user)` in the same class

File: app/Http/Controllers/Auth/RegisterController.php [hunk 2]
```diff
@@ -56,26 +65,34 @@ protected function validator(array $data)
...
         ]);
     }
 
-    /**
-     * Create a new user instance after a valid registration.
-     *
-     * @param  array  $data
-     * @return User
-     */
-    protected function create(array $data)
```
Comment (nabeelio) [inline app/Http/Controllers/Auth/RegisterController.php]: Leave this as create, it follows the Laravel convention for the POST processing

File: app/Services/PilotService.php [hunk 3]
```diff
@@ -55,4 +55,9 @@ function () {
         return $pilot;
     }
 
+    public function create()
```
Comment (nabeelio) [inline app/Services/PilotService.php]: Yeah move the actual registration code, after the `validate()` and above the view stuff into here. Then we can send the validation email, etc from here as well.

**Comment by Vansers** [comment]
Oh shoot, I forgot about moving that stuff into service. I'll fix that up and report back. Thanks!

**Comment by Vansers** [comment]
Umm, silly question, how would I call the the function create() in the Register Controller?

**Comment by nabeelio** [comment]
@Vansers it's automatic - https://laravel.com/docs/5.4/controllers#resource-controllers

**Comment by Vansers** [comment]
@nabeelio Fixed as your request. ;-)

**Comment by Vansers** [comment]
Added error pages for 404 and 401. 503 was already added.

**Comment by Vansers** [comment]
#63

**Pull request merged**